docs/user-guide/global-configurations/git-accounts.md

```diff
@@ -1,10 +1,10 @@
 # Git Accounts
 
-Git Accounts allow you to connect your code source with Devtron. You’ll be able to use these git accounts to build code using the CI pipeline.
+Git Accounts allow users to connect your code source with Devtron. Users be able to use these git accounts to build code using the CI pipeline.
```

Comment:
Use you/your instead of user/users. Using you/your brings a conversational aspect to the documentation as the user him/herself is reading the document. Pls connect regarding this.